src/configmanager.cpp

```diff
@@ -328,7 +328,8 @@ bool ConfigManager::getBoolean(boolean_config_t what) const
 float ConfigManager::getExperienceStage(uint32_t level) const
 {
 	auto it = std::find_if(expStages.begin(), expStages.end(), [level](ExperienceStages::value_type stage) {
```

Comment:
While at that you can change it here to use `auto`
```suggestion
	auto it = std::find_if(expStages.begin(), expStages.end(), [level](auto stage) {
```

And I THINK this is possible:
```suggestion
	auto it = std::find_if(expStages.begin(), expStages.end(), [level](auto&& [minLevel, maxLevel]) {
```
or
```suggestion
	auto it = std::find_if(expStages.begin(), expStages.end(), [level](auto stage) {
            auto&& [minLevel, maxLevel] = stage;
```

Reply:
feel free to commit on this branch (please)